app/admin/admin_users.rb

```diff
@@ -0,0 +1,28 @@
+ActiveAdmin.register AdminUser do
+  permit_params :email, :password, :password_confirmation
+
+  index do
+    selectable_column
+    id_column
+    column :email
+    column :current_sign_in_at
+    column :sign_in_count
+    column :created_at
+    actions
```

Comment:
So that  adminUsers can be edited, deleted and viewed by clicking on buttons
![Screenshot from 2022-03-01 17-11-54](https://user-images.githubusercontent.com/57142378/156163224-f57531ab-e362-4ae1-b327-a3382ba63690.png)